src/ocean/io/select/client/SelectEvent.d

```diff
@@ -251,6 +248,7 @@ public class SelectEvent : ISelectEvent
 
     protected override bool handle_ ( ulong n )
     {
+        verify(this.handler !is null);
```

Comment:
this one is added here, because it was in line 147, but never compiled (in contracts of the abstract method's implementations are not compiled).